src/config/dynamic-theme-fixes.config

```diff
@@ -13725,6 +13725,15 @@ g#ubuntu-logo > path
 
 udemy.com
 
+CSS
+.radio:not(.radio-button) input[type="radio"]:checked + .radio-label::before, .radio-inline:not(.radio-button) input[type="radio"]:checked + .radio-label::before {
+    background-color: rgb(93, 122, 136);
+    box-shadow: rgb(27, 29, 29) 0px 0px 0px 2px inset;
+}
+div[class^='curriculum-item-view--scaled-height-limiter'], div[class*=' curriculum-item-view--scaled-height-limiter'] {
```

Comment:
Ehm, we should dark reader change the max-height?

Reply:
my 2560x1080 (ultra wide) screen

### without the change

![Screenshot_20211105_092014](https://user-images.githubusercontent.com/34714622/140510443-bc841751-e896-4e21-81d8-a1246cc84fda.png)


### with the change, the space of the content is for the questions is proportional to the screen 

![Screenshot_20211105_091926](https://user-images.githubusercontent.com/34714622/140510490-fb6bacb2-3d85-4fde-a843-ed1fe0bf2c61.png)

Reply:
Is that a bug that Dark Reader caused? Otherwise it isn't a fix, just a cosmetic change.

Reply:
```we should dark reader change the max-height?```


for me the change makes sense, but I think the site that should fit my screen

dark reader is more for theme, not layout. but it has helped me a lot at this point

the decision is yours ;)

if not added by you, I'll have to keep putting it in the dynamic theme every time (it's not saving the settings)

Reply:
> Is that a bug that Dark Reader caused? Otherwise it isn't a fix, just a cosmetic change.

not a bug

Reply:
> for me the change makes sense, but I think the site that should fit my screen

I get that, but this config is exclusively for fixes that is caused by Dark Reader.